.github/workflows/CD.yml

```
@@ -14,14 +14,9 @@
 name: C/D
 
 on:        
-  workflow_dispatch:
-    inputs:
-      version:
-        description: 'the version number to release'
-        required: true
-      sha:
-        description: 'the github sha to release'
-        required: true
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+*"
```

```suggestion
      - "?v[0-9]+.[0-9]+.[0-9]+*"
```
Do we need this to indicate start of string?

---

Hmmm I'm not sure but this is the pattern than upstream repositories use

---

Nvm I think I'm off base here since this is glob matching and not regex. See [this](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet) cheat sheet for ref.